webapp/channels/src/components/logged_in/index.ts

```diff
@@ -46,13 +44,14 @@ function mapStateToProps(state: GlobalState, ownProps: Props) {
 }
 
 // NOTE: suggestions where to keep this welcomed
-const getChannelURLAction = (channel: Channel, teamId: string, url: string) => (dispatch: DispatchFunc, getState: () => GlobalState) => {
+const getChannelURLAction = (channelId: string, teamId: string, url: string) => (dispatch: DispatchFunc, getState: () => GlobalState) => {
```

Comment:
This function is literally only used for this Desktop App feature as far as I can tell. The Desktop App doesn't need the whole channel object and I think it's simpler to avoid passing it all up, so I changed it here.

Reply:
Not for this PR, but it would been nice if we grouped or marked these functions that are only used by the desktop API. I guess that's what that comment above the function was meant to do 😛

Reply:
Going forward, we might want to make a bigger Desktop App module to have:
1) The main module containing the name/version, as well as any helpers to talk to the API/do backwards compatibility
2) The individual wrapped API calls
3) Any other Desktop-specific code (which is where I'd imagine that function would go)

We COULD setup the framework for that here, and then start moving stuff over later. 1/5 on that.